From 90212b90dd37bf895d098bb912e635af122fffe8 Mon Sep 17 00:00:00 2001
From: Quentin Quadrat <lecrapouille@gmail.com>
Date: Sat, 6 Nov 2021 13:03:40 +0100
Subject: [PATCH] Do not care to be root when inside docker

---
 script/bootstrap.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/script/bootstrap.py b/script/bootstrap.py
index 4b4d2da..729de18 100755
--- a/script/bootstrap.py
+++ b/script/bootstrap.py
@@ -48,9 +48,9 @@ def parse_args():
 def check_root():
     if os.geteuid() == 0:
         print "We suggest not running this as root, unless you're really sure."
-        choice = raw_input("Do you want to continue? [y/N]: ")
-        if choice not in ('y', 'Y'):
-            sys.exit(0)
+        #choice = raw_input("Do you want to continue? [y/N]: ")
+        #if choice not in ('y', 'Y'):
+        #    sys.exit(0)
 
 
 def setup_python_libs():
-- 
2.30.2

